gubernator/log_parser.py

```diff
@@ -56,14 +56,13 @@ def log_html(lines, matched_lines, hilight_words, skip_fmt):
             output.append('<span class="skip">%s</span>' % skip_fmt(skip_amount))
         elif skip_amount > 1:
             skip_id = 'skip_%s' % match
-            output.append('<span class="skip"><a href="javascript:show_skipped(\'%s\')"'
-                % skip_id)
-            output.append('onclick="this.style.display=\'none\'">%s</a></span>'
-                % skip_fmt(skip_amount))
-            output.append('<div id="%s" style="display:none;"><p><span class="skipped">'
-                % skip_id)
+            skip_string = ("<span class=\"skip\"><a href=\"javascript:show_skipped(\'%s\')\""
```

Comment:
this would scan easier if you used Python's implicit string concatenation:

    skip_string = ('<span class="skip">'
        '<a href="javascript:show_skipped(\'%s\')"'
        ...